qiskit_optimization/runtime/vqe_program.py

```diff
@@ -222,6 +222,19 @@ def wrapped_callback(*args):
         else:
             return None
 
+    @classmethod
+    def supports_aux_operators(cls) -> bool:
+        """Whether computing the expectation value of auxiliary operators is supported.
+
+        If the minimum eigensolver computes an eigenstate of the main operator then it
+        can compute the expectation value of the aux_operators for that state. Otherwise
+        they will be ignored.
+
+        Returns:
+            True if aux_operator expectations can be evaluated, False otherwise
+        """
```

Comment:
I think you don't have to add this docstring if it's the same as in the `MinimumEigensolver` base class, since it will just be inherited in the documentation